README.md

```diff
@@ -16,6 +46,72 @@ export default () => {
 };
 ```
 
-## Authorization
+### Authorization
+
+Authorization will work with hosted GraphQL services such as scaphold and graph.cool. This works by passing a stored authentication token inside of the authorization header. This token is currently assumed to stored in a cookie and cookies by the value provided in `ApolloClientAuthKeyToken` (defaults to "token").
+
+---
+
+### API
+
+#### Registration API
+
+##### ApolloClientEndpointToken
+
+```js
+import {ApolloClientEndpointToken} from 'fusion-apollo';
+```
+
+A token with the GraphQL endpoint which the Apollo HttpLink client communicates with. This can be an absolute path to a local GraphQL server, or a remote hosted GraphQL server.
+
+###### Type
+
+```js
+type ApolloClientAuthKeyToken = string;
```

Comment:
_(Minor)_ Technically `Token<string>`.  Might be more correct to say `type ApolloCluentAuthKey = string;`, although I'm not sure if we need this if we also specify that it's a string right below here, especially given that this is a primitive type.

Reply:
Thanks. Going to just delete these for now and keep the description because it's a primitive type and that feels less confusing.